custom_components/zwave_mqtt/__init__.py

```diff
@@ -21,6 +21,7 @@
 from .const import DATA_NODES, DATA_VALUES, DOMAIN, PLATFORMS, TOPIC_OPENZWAVE
 from .discovery import DISCOVERY_SCHEMAS, check_node_schema, check_value_schema
 from .entity import ZWaveDeviceEntityValues
+from .services import ZWaveServices
```

Comment:
Can we break the services part out into a new PR? That way it's easier to review.

Reply:
No problem, will do some git sorcery to get those commits into a seperate branch. Thanks for reviewing so far!